Unsorted/24299388_172c77c.txt

```diff
@@ -25,4 +31,9 @@
 		いっぱいいるから<end_line>
 		出ちゃいけないんだよ<end_line>
 	</sjis>
+	<ascii>
+		You shouldn't wander<end_line
+		outside the village,<end_line>
+		there are many stray summons out there<end_line>
```

Comment:
Line is too long (38 chars vs 36 char limit). Considering using "strays" over the lengthy stray summons.

In addition, stray summons is a special term, so I've been capitalizing it as "Stray Summons". Whether or not you want to, we should at least keep it consistent.

Reply:
In Scs2 the term is sometimes shortened as "Strays" so you might want to use that too...

![image 1](https://user-images.githubusercontent.com/7575105/36377577-7740f2d2-1556-11e8-98e7-6f20b56cf4f0.png)

Reply:
Changed it to just "strays" 👍  c0b273b